launcher/BUILD.bazel

```diff
@@ -24,12 +26,18 @@ cc_library(
 )
 
 cc_binary(
-    name = "jazzer",
+    name = "jazzer_single_arch",
     linkstatic = True,
-    visibility = ["//visibility:public"],
+    tags = ["manual"],
     deps = [":jazzer_main"],
 )
 
+universal_binary(
```

Comment:
Could you add a comment regarding non-macOS platforms? I was a somewhat confused and a comment could help the next reader along. The docs state that `universal_binary` would just create a symlink to the input binary.

Reply:
Added a comment.